moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -445,6 +445,10 @@ void RobotState::setJointEfforts(const JointModel* joint, const double* effort)
 
 void RobotState::setJointGroupPositions(const JointModelGroup* group, const double* gstate)
 {
+  if (group == nullptr)
```

Comment:
Please check  ```if ()``` format in This PR.

Reply:
There is no similar writing in the source, so I will fix it.

Reply:
OK